### ISSUE TITLE
Printing training progress

### ISSUE THREAD

**Issue by druefena** [lRomul/ball-action-spotting#3]
Hello,

First of all, thanks a lot for this great repo and detailed insights into your approach to training ball actions.

I have a question related to the training script. I would like to print training progress, e.g.:
Epoch 1/20
224/768 [========] ------------------------  264us/step - loss: 0.4730 - acc: 0.7747

I understand you are using your own training library (argus), which does some things in train_step I don't quite get. Can you please suggest how this can be done? (wrapping tqdm around the data loader does not work).

Thanks a lot in advance.

**Comment by lRomul** [comment]
Hello, thank you!

You need a custom callback for logging. I think you can use the tqdm callback that has been implemented in another solution with argus:
https://github.com/sankovalev/deep_chimpact.drivendata/blob/master/src/modules/callbacks.py

If you want to understand how it works or improve it, then here is the documentation page about callbacks:
https://pytorch-argus.readthedocs.io/en/latest/api_reference/callbacks/base_callback.html

**Comment by druefena** [comment]
Thanks a lot for the quick and useful answer. The TQDM callback is a good starting point, I just added the total=len(state.data_loader) on_epoch_start so that I get the desired progress bar.
